{"uuid": "a73513b7-1c62-4e18-b162-773cfcdd6606", "children": ["6f6f2254-11a4-40bb-991d-03f1008eee07"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "NameError: name 'EC' is not defined\n", "trace": "  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\cygwin64\\home\\Viktoriya\\Practikum_repos\\Sprint_6\\tests\\conftest.py\", line 12, in driver\n    EC.presence_of_element_located((By.TAG_NAME, \"body\"))\n    ^^\n"}, "start": 1743949416057, "stop": 1743949429721}], "start": 1743949416057, "stop": 1743949429931}